examples/native_networking/minimqtt_adafruitio_native_networking.py

```diff
@@ -59,6 +60,14 @@ def message(client, topic, message):
 
 # Create a socket pool
 pool = socketpool.SocketPool(wifi.radio)
+ssl_context = ssl.create_default_context()
+
+# If you need to use certificate/key pair authentication (e.g. X.509), you can load them in the
+# ssl context by uncommenting the line below
```

Comment:
@zbauman3 Might also want to specify in your comment that these two files should be located within the secrets file's `device_cert_path` and `device_key_path`

Reply:
Will do! AFK right now, I'll update and tag you tomorrow 👍

Reply:
@brentru I added some more detail to the comments. Thanks for the suggestion!